packages/seacas/libraries/ioss/src/Ioss_CodeTypes.h

```diff
@@ -37,9 +37,9 @@ inline const std::string IOSS_SYM_TENSOR() { return std::string("sym_tensor_33")
 #define SEACAS_HAVE_MPI
 /* #undef IOSS_THREADSAFE */
 /* #undef SEACAS_HAVE_KOKKOS */
-/* #undef SEACAS_HAVE_DATAWAREHOUSE */
 #define SEACAS_HAVE_EXODUS
 #define SEACAS_HAVE_CGNS
+#define SEACAS_HAVE_FAODEL
```

Comment:
This block is if we have FAODEL activated in Sierra.  This should not be defined until FAODEL is added as a TPL in Sierra

Reply:
Fixed in commit 7ac8f39.